hr_timesheet_improvement/README.rst

```diff
@@ -0,0 +1,47 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+Timesheet improvements
+======================
+
+Modifies timesheet behavior:
+ * Ensure a DESC order on timesheet lines
+ * Set default date for manually entering attendance to max attendance date
+ * Redefine constraint on timesheets to check alternation of 'sign in' and
+   'sign out' only on current timesheet instead of doing it on all timesheets
+   of the employee
+
```

Comment:
Can you add the 'Usage' section with the new 'Try me on runbot' button https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst#usage